rssemitter.go

```diff
@@ -52,7 +52,7 @@ func (r *RssFeedEmitter) List() *[]gofeed.Feed {
 // Remove expects a string and removes the feed from the list
 func (r *RssFeedEmitter) Remove(url string) {
 	for i := 0; i < len(r.feeds); i++ {
```

Comment:
Is this possible to change this in a for range loop? 🤔

Reply:
Is a for range control structure faster performance-wise?

Reply:
Maybe 🤔 but since we're iterating through an array is it best to use for range? Or no? My though, what do you think bro @kenkoii ? 🤔

Reply:
I think the underlying implementation of range is still a simple c for loop. But for brevity we can change it to Range. It makes it more readable.

Reply:
Yah 👍 thanks @kenkoii 🍕

Reply:
I think we should stick with for loop until I do some research. I'll create a separate issue if I find anything regarding performance.

Reply:
Okay noted @kenkoii 👍 will merge this one then..